src/graphapi/engine.cpp

```diff
@@ -36,6 +43,179 @@ GraphPatternExecutor::~GraphPatternExecutor() = default;
 
 size_t GraphExecutorFind20::getWorkspaceSize() const { return mSolution.GetWorkspaceSize(); }
 
+namespace {
+
+const std::string_view base64Chars = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+                                     "abcdefghijklmnopqrstuvwxyz"
+                                     "0123456789+/";
+
+std::string base64Encode(const char* data, size_t length)
```

Comment:
Should we move base64 encode/decode to utils of any kind? Looks like it might be commonly used routine...

Reply:
Why are the pointer and the size unbundled here? Can we pass `vector<char>`
Especially if it's inside an anonymous namespace and for a single specific case in that PR.

But probably for a common case, like Vsevolod mentioned, it's worth to have those fields separated.

Reply:
Among possible options:
```C++
std::string base64Encode(const uint8_t* begin, const uint8_t* end)
std::string base64Encode(const uint8_t* begin, size_t size)
std::string base64Encode(const std::vector<uint8_t>& data)
```
I chose that one, but probably better to provide all of them.